### ISSUE TITLE
Futures SubscribeToUserDataUpdates -> Issue

### ISSUE THREAD

**Issue by ivanfiao** [JKorf/Binance.Net#794]
When using FutureCoin.SubscribeToUserDataUpdates socket client, the library shows the spent time to process Socket message when LogVerbosity.Debug is on:

Ex.:
Socket 1 message processed in 2ms
Socket 1 message processed in 15ms
Socket 1 message processed in 30ms

It seems that the code is taking to much time to process this socket messages. Sometimes the library processes the messages in 2-3 ms, but sometimes it takes 30ms to process. I'm trying to develop a code that is latency sensitive, so for me 5ms make some difference to my performance. 

Since the socket message processing is basically a JSON parse, it should take 1-2ms maximum to process the message.

![WhatsApp Image 2021-07-01 at 22 03 06](https://user-images.githubusercontent.com/61050503/124308309-7f4b5c80-db3f-11eb-9178-8def7c20ef6c.jpeg)

**Comment by JKorf** [comment]
Hi, when just running the `SubscribeToAllBookTickerUpdatesAsync` subscription, which is a high traffic subscription, with  an empty handler I get processing times of 0ms. I would suggest that the slowing down happens during your implementation of the handling of the message.

**Comment by ivanfiao** [comment]
Hi Jkorf,

Thank you for your reply.

I've made the same test with SubscribeToAllBookTickerUpdatesAsync and the message were processed in 0 ms, as you pointed out.

The problem is when using SubscribeToUserDataUpdates, for some reason the code is taking too much time do proccess. For testing purposes I've made a clean code that only subscribe to the websocket, then I'd post and cancel orders manually, and still the long processing time persists.

both, code and latency can be checked out in the print below.

![image](https://user-images.githubusercontent.com/61050503/124332254-74a4bd80-db67-11eb-858b-b545dee47e46.png)
